### ISSUE TITLE
WASD and Flying Issue 

### ISSUE THREAD

**Issue by stevel4857** [ubik2/aframe-network-component#1]
When importing a gltf as a building the environment no longer lets me fly. Not sure what's going on. 

http://www.cablecenter.org/vr/network/
`<!DOCTYPE html>
<html>
  <head>
<script src="https://aframe.io/releases/0.7.0/aframe.min.js"></script>
<script src="https://cdn.rawgit.com/delapuente/aframe-sharedspace-component/master/dist/aframe-sharedspace-component.min.js"></script>
<script src="https://rawgit.com/feiss/aframe-environment-component/master/dist/aframe-environment-component.min.js"></script>
</head>
<body>

<a-scene look-controls wasd-controls="fly: true">
<a-assets>
  <a-mixin id="user" visible="false"  share="position, rotation"></a-mixin>
<a-asset-item id="head" src="scene.gltf"></a-asset-item>
</a-assets>
    <a-entity sharedspace="hold: true; audio: true" avatars="onmyself: user">
    <a-entity environment="preset: forest" id="forest"><a-entity gltf-model="#head" id="building" position="-54.83 1.67 58.71" scale="0.020 0.020 0.020" rotation="-1.904 .005 .005"></a-entity><a-entity>
  </a-entity>
</a-scene>
<template>
  <a-entity>
    <a-sphere radius="0.1" color="#ffffff"></a-sphere>
    <a-sphere position="0.05 0.03 -0.08" radius="0.02" segments-width="8" segments-height="8" color="#000000"></a-sphere>
    <a-sphere position="-0.05 0.03 -0.08" radius="0.02" segments-width="8" segments-height="8" color="#000000"></a-sphere>
    <a-sphere class="themable" position="0 -0.07 -0.1" scale="1 1 0.5" segments-width="4" segments-height="4" radius="0.02" color="#11fd3e"></a-sphere>
    <a-cone class="themable" position="0.03 -0.07 -0.1" rotation="0 0 90" scale="1 1 0.5" segments-radial="8" segments-height="1" height="0.03" radius-bottom="0.03"  color="#1cff3c"></a-cone>
    <a-cone class="themable" position="-0.03 -0.07 -0.1" rotation="0 0 -90" scale="1 1 0.5" segments-radial="8" segments-height="1" height="0.03" radius-bottom="0.03"  color="#1cff3c"></a-cone>
  </a-entity>
</template>
<script>
  var scene = document.querySelector('a-scene');
      
  (function start() {
    if (!scene.hasLoaded) {
      scene.addEventListener('loaded', start);
      return;
    }
        
    // Now it’s safe to change the scene components...

var prefix = window.location.host.split('.')[0] + '-';
var currentUrl = new URL(window.location);
var roomName = currentUrl.search.substr(1);

if (!roomName) {
  roomName = prefix + Date.now();
  currentUrl.search = roomName;
  history.pushState({}, '', currentUrl.href);
}
        
var room = document.querySelector('[sharedspace]');
room.setAttribute('sharedspace', { room: roomName, hold: false });

  }());
</script>
</body>
</html>`

**Comment by ubik2** [comment]
I don't believe this is related to the network component (which is intended to synchronize movement of multiple clients).
Your scene has two objects with the wasd-controls component. One of them has fly set to true, but the other lacks that setting. I suspect the latter's setting is being used.
I modified my example scene (to set the wasd-controls="fly: true"), and after making that modification, I'm able to fly.
